### PR TITLE
Fix: Fix process bug in laser and cnc

### PR DESCRIPTION
**Pull request by bbaa77770**
Supply LU-714  SB-486
And some other bug

### CODE REVIEW

File: src/app/widgets/CncLaserList/ToolPathList/ToolPathListBox.jsx [hunk 1]
```diff
@@ -35,6 +39,47 @@ class ToolPathListBox extends PureComponent {
     contextMenuRef = React.createRef();
 
     actions = {
+        getContent: (headType, toolPath) => {
```
Comment (meiliang1123) [inline src/app/widgets/CncLaserList/ToolPathList/ToolPathListBox.jsx:42]: renderDom should be method of component like:
`renderToolTip(headType, toolPath){
}`

File: src/app/widgets/CncLaserList/ToolPathList/ToolPathListBox.jsx [hunk 2]
```diff
@@ -35,6 +39,47 @@ class ToolPathListBox extends PureComponent {
...
+            console.log('tp', toolPath);
```
Comment (meiliang1123) [inline src/app/widgets/CncLaserList/ToolPathList/ToolPathListBox.jsx:43]: remove console.log

File: src/app/widgets/PrintingConfigurations/Configurations.jsx [hunk 3]
```diff
@@ -187,11 +187,7 @@ class Configurations extends PureComponent {
                                                 )}
                                                 { type === 'bool' && (
                                                     <td>
-                                                        <div
-                                                            className={
-                                                                classNames(styles[defaultValue ? 'checked' : 'unchecked'])
-                                                            }
-                                                        />
+                                                        {defaultValue ? i18n._('Yes') : i18n._('No')}
```
Comment (meiliang1123) [inline src/app/widgets/PrintingConfigurations/Configurations.jsx:190]: change checkbox to string?

**Review by meiliang1123 — CHANGES_REQUESTED** [review]

File: src/app/widgets/PrintingConfigurations/Configurations.jsx (hunk 3 above)
Comment (bbaa77770) [reply src/app/widgets/PrintingConfigurations/Configurations.jsx:190]: Yep

**Pull request merged**